apps/pie-docs/src/assets/styles/components/_card.scss

```diff
@@ -0,0 +1,88 @@
+@use '../fozzie-config' as f;
+
+$card-max-width: 328px;
+$card-image-height: 180px;
+$card-label-height: 72px;
+
+.c-card-wrapper {
+    container-type: inline-size;
+}
+
+.c-card-list {
+    margin: f.$spacing-f 0;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: f.$spacing-e;
+    list-style: none;
+    padding: 0;
+}
+
+@container (min-width: 680px) {
```

Comment:
out of interest can this be added into the block above much like media queries in sass?

Reply:
it does 🥳